README.md

```diff
@@ -1,24 +1,185 @@
+## 🛠 Built With <a name="built-with"></a>
 
-* Deployment instructions
+### Tech Stack <a name="tech-stack"></a>
 
-* ...
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby on rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">postgres</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+- Ruby on Rails
+- PostgreSQL Database
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> None
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> To start and run this project
+```
+bundle install
+```
+```
+ruby main.rb
+```
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+- Visual Studio Code.
+- Git Flow.
+- PostgreSQL
+
+
+### Install & Setup
+
+To setup and install this sample ROR Blog project, follow the below steps:
+- Clone this project by the command: 
+
+```
+$ git clone https://github.com/chuaindia/ror_capstone_buget_app.git
+```
+
+- Then switch to the project folder by the bellow query:
+
+```
+$ cd ror_capstone_buget_app
+```
+
+- Install the required dependecies with `bundle install`
+- Install npm dependencies with `npm install`
+- **Before running below commands update [config/database.yml](./config/database.yml) with your local `DB`credentials. Then run:**
+      - **`rake db:create`**
+      - **`rake db:migrate`**
+     
+- With Ruby on Rails and PostgreSQL installed in your environment, you can run `rails server` to run the website.
+
+### Tests
+
+- Run `rake db:seed RAILS_ENV=test` to seed the testing database
+- Run `rspec`
+
+### Deployment
+
+- I will deploy the project with [render](https://render.com/docs/deploy-rails), after which you can access the live version of the project in the Live Demo section. I   tried it several times following the process explained but it failed.The link for failed deployment is attached : https://ror-capstone-budget-app.onrender.com.
+- I tried deployment with netlify : The details is attached in the pull request.
+
+### Video Presentation
+
+- I made a loom video presentation . The link is : https://www.loom.com/share/8cb3175e5c07456a8e902d6bd9639749
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Tanusri Ghosh**
+- GitHub: [@chuaindia](https://github.com/chuaindia)
+- Twitter: [@chuaghosh](https://twitter.com/chuaghosh25) 
+- LinkedIn: [@Ghosh](https://www.linkedin.com/in/tanusri-ghosh-2a56b814/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
```

Comment:
- [ ] Kindly remove this title description to make your readme look clean and descriptive :blush: These section descriptions are just guidelines from the owner of the repository for just one time when he/she will insert data into his/her readme file :+1: These are unnecessary for the viewers of that repository

**Section Description** :arrow_down:

```
> Describe 1 - 3 features you will add to the project.
```